docs/02-quick-start/01-installation/README.mdx

```diff
@@ -8,4 +8,60 @@ import DocCardList from '@theme/DocCardList';
 
 This section provides instructions to quickly install an Ozone instance on your local machine that can be used for testing and evaluation. For instructions to install a production Ozone instance, see the [Administrator Guide](../../administrator-guide/installation).
 
+<div className="container">
```

Comment:
Currently, this causes the page to have some leftover link to the subpages.

<img width="2118" height="1438" alt="image" src="https://github.com/user-attachments/assets/a4a304d8-dd21-44d0-ab01-425b78aac1d4" />

Can we disable the subpages link for the quick start so the bottom links can be removed?